app/views/overrides/organizations/_edit_override.html.erb

```diff
@@ -8,11 +8,21 @@
     <%= selectable_f(f, :service_level, options_for_select(service_level_options, service_level_selected), {}, :label => _('Default System SLA')) %>
   <% end %>
 
+
   <%= field(f, _('Debug Certificate'),
             :help_inline => _('This certificate allows a user to view the repositories in any environment from a browser.')) do
     button_tag _('Generate and Download'), :id => :download_debug_cert_key,
                :type => :button, :style => 'margin-top: 8px;', :class => 'btn btn-small',
                'data-url' => "/katello/api/v2/organizations/#{@taxonomy.id}/download_debug_certificate"
   end %>
 
-<% end%>
+  <% if @taxonomy.manifest_imported? %>
```

Comment:
Thinking about this a bit more. On the manifest modal manifest actions like this are disabled if the upstream subscriptions controller 'ping' api fails. it will fail under circumstances like:

- no manifest imported
- the system is in 'disconnected' mode
- the manifest is expired

If we can be consistent that is ideal. A way to achieve this is to override the `edit` method (assuming that's what it's called) in the same controller you overrode `update` and provide a new instance variable for the view `@can_toggle_sca` which will be true/false determined by code similar to here https://github.com/Katello/katello/blob/master/app/controllers/katello/api/v2/api_controller.rb#L215-L224